contents/handbook/small-teams/website-docs/index.mdx

```diff
@@ -55,13 +50,37 @@ If a screenshot is posted directly to a Github issue, that's a great place for f
 
 Some of the design tools we use, like Balsamiq and Figma, both have built-in commenting, which is useful for prototypes with multiple pages. If we provide a link to a prototype in one of these tools, please leave a comment using the app's comment system. This helps us review and take action on comments later, and creates a single place for discussion around a particular topic.
 
-**Important:** We prioritize feedback based on alignment with business goals. _Everyone has feedback about design._ (If feedback is more of a personal opinion than a business-related perspective, we’ll note it, but don't be offended if your feedback isn't specifically addressed!)
+**Important:** We prioritize feedback based on alignment with business goals. _Everyone has feedback about design._ (If feedback is more of a personal opinion than a business-related perspective, we’ll note it, but please don't be offended if your feedback isn't specifically addressed!)
+
+## Working with us
+
+If you're in a team that isn't Website & Docs, and you want to make a change to the website, here is a guide to working with us.
+
+Most helpful - objective changes:
+
+* Fixing errors
+* Adding missing functionality
+* Correcting typos
+* Fixing broken links
+* Posting an issue about a concern
+
+Less helpful / can be net negative - subjective changes:
+
+* CTA changes / additions
+* Wording changes, especially titles on the most prominent pages
+* Menu / navigation structure changes
+* Aesthetic changes
+* Deleting prominent pages / large blocks of content
```

Comment:
Is it the case that you'd prefer an issue > PR for this? 

I'm just thinking of objective changes (e.g. I recently needed to delete some self-hosted landing pages) and my goto route is to make a PR and tag Eli/Cory for review. But I'm reading this and it sounds like you'd prefer an issue to highlight the concern instead?